File: taskmanager/src/androidTest/java/com/example/alexeyglushkov/taskmanager/task/TestTaskProvider.java
package com.example.alexeyglushkov.taskmanager.task;

import android.os.Handler;

import com.example.alexeyglushkov.taskmanager.task.PriorityTaskProvider;

/**
 * Created by alexeyglushkov on 30.08.15.
 */
public class TestTaskProvider extends PriorityTaskProvider {
    public TestTaskProvider(Handler handler, String id) {
        super(handler, id);
    }


}

File: taskmanager/src/test/java/com/example/alexeyglushkov/taskmanager/task/TestTaskProvider.kt
package com.example.alexeyglushkov.taskmanager.task

import com.example.alexeyglushkov.taskmanager.providers.PriorityTaskProvider
import com.example.alexeyglushkov.taskmanager.runners.ThreadRunner

/**
 * Created by alexeyglushkov on 30.08.15.
 */
open class TestTaskProvider(threadRunner: ThreadRunner, id: String) : PriorityTaskProvider(threadRunner, id)
